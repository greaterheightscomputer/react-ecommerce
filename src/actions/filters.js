// SET_TEXT_FILTER  
export const setTextFilter = (text = '') => ({
    type: 'SET_TEXT_FILTER',
    text
});
// SORT_BY_DATE
export const sortByDate = () => ({
    type: 'SORT_BY_DATE' 
});    
// SORT_BY_AMOUNT    
export const sortByAmount = () => ({
    type: 'SORT_BY_AMOUNT'
});
//CategoryType
export const categoryTypeMen = () => ({
    type: 'CATEGORY_TYPE_MEN'    
}); 
export const categoryTypeWomen = () => ({
    type: 'CATEGORY_TYPE_WOMEN'
});
export const categoryTypeBoy = () => ({
    type: 'CATEGORY_TYPE_BOY'
});
export const categoryTypeGirl = () => ({
    type: 'CATEGORY_TYPE_GIRL'
});
//itemType - Men
export const itemTypeMenFootWear = () => ({
    type: 'ITEM_TYPE_MEN_FOOTWEAR'
});  
export const itemTypeMenClothe = () => ({   
    type: 'ITEM_TYPE_MEN_CLOTHE'     
}); 
export const itemTypeMenHeadWear = () => ({
    type: 'ITEM_TYPE_MEN_HEADWEAR'
});  
export const itemTypeMenUnderWear = () => ({
    type: 'ITEM_TYPE_MEN_UNDERWEAR'
}); 
export const itemTypeMenBag = () => ({
    type: 'ITEM_TYPE_MEN_BAG'
}); 
export const itemTypeMenTie = () => ({
    type: 'ITEM_TYPE_MEN_TIE'
}); 
export const itemTypeMenBelt = () => ({
    type: 'ITEM_TYPE_MEN_BELT'
}); 
export const itemTypeMenJewellery = () => ({
    type: 'ITEM_TYPE_MEN_JEWELLERY'
}); 
//itemType - Women
export const itemTypeWomenFootWear = () => ({
    type: 'ITEM_TYPE_WOMEN_FOOTWEAR'
});  
export const itemTypeWomenClothe = () => ({   
    type: 'ITEM_TYPE_WOMEN_CLOTHE'     
}); 
export const itemTypeWomenHeadWear = () => ({
    type: 'ITEM_TYPE_WOMEN_HEADWEAR'
});  
export const itemTypeWomenUnderWear = () => ({
    type: 'ITEM_TYPE_WOMEN_UNDERWEAR'
}); 
export const itemTypeWomenBag = () => ({
    type: 'ITEM_TYPE_WOMEN_BAG'
}); 
export const itemTypeWomenTie = () => ({
    type: 'ITEM_TYPE_WOMEN_TIE'
}); 
export const itemTypeWomenBelt = () => ({
    type: 'ITEM_TYPE_WOMEN_BELT'
}); 
export const itemTypeWomenJewellery = () => ({
    type: 'ITEM_TYPE_WOMEN_JEWELLERY'
}); 
//itemType - Boy
export const itemTypeBoyFootWear = () => ({
    type: 'ITEM_TYPE_BOY_FOOTWEAR'
});  
export const itemTypeBoyClothe = () => ({   
    type: 'ITEM_TYPE_BOY_CLOTHE'     
}); 
export const itemTypeBoyHeadWear = () => ({
    type: 'ITEM_TYPE_BOY_HEADWEAR'
});  
export const itemTypeBoyUnderWear = () => ({
    type: 'ITEM_TYPE_BOY_UNDERWEAR'
}); 
export const itemTypeBoyBag = () => ({
    type: 'ITEM_TYPE_BOY_BAG'
}); 
export const itemTypeBoyTie = () => ({
    type: 'ITEM_TYPE_BOY_TIE'
}); 
export const itemTypeBoyBelt = () => ({
    type: 'ITEM_TYPE_BOY_BELT'
}); 
export const itemTypeBoyJewellery = () => ({
    type: 'ITEM_TYPE_BOY_JEWELLERY'
}); 
//itemType - Girl
export const itemTypeGirlFootWear = () => ({
    type: 'ITEM_TYPE_GIRL_FOOTWEAR'
});  
export const itemTypeGirlClothe = () => ({   
    type: 'ITEM_TYPE_GIRL_CLOTHE'     
}); 
export const itemTypeGirlHeadWear = () => ({
    type: 'ITEM_TYPE_GIRL_HEADWEAR'
});  
export const itemTypeGirlUnderWear = () => ({
    type: 'ITEM_TYPE_GIRL_UNDERWEAR'
}); 
export const itemTypeGirlBag = () => ({
    type: 'ITEM_TYPE_GIRL_BAG'
}); 
export const itemTypeGirlTie = () => ({
    type: 'ITEM_TYPE_GIRL_TIE'
}); 
export const itemTypeGirlBelt = () => ({
    type: 'ITEM_TYPE_GIRL_BELT'
}); 
export const itemTypeGirlJewellery = () => ({
    type: 'ITEM_TYPE_GIRL_JEWELLERY'
});
// SET_START_DATE   
export const setStartDate = (startDate) => ({
    type: 'SET_START_DATE',
    startDate
});
// SET_END_DATE
export const setEndDate = (endDate) => ({
    type: 'SET_END_DATE',
    endDate
});