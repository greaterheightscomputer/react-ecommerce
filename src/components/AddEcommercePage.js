import React from 'react';

const AddEcommercePage = () => (
    <div>
        This is from my add expense component
    </div>
);

export default AddEcommercePage;