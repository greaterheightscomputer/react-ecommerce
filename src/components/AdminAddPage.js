import React from 'react';
import { connect } from 'react-redux';
import AdminForm from './AdminForm';
import { startAddEcommerce } from '../actions/ecommerces';

export class AdminAddPage extends React.Component {
    onSubmit = (ecommerce) => {
        // console.log(ecommerce)
        // props.dispatch(addEcommerce(ecommerce));
        this.props.startAddEcommerce(ecommerce);
        this.props.history.push('/admin_dashboard');
    };
    render() {
        return (
            <div className="list-body">
                <div className="page-admin-header">
                    <div className="content-container">
                        <h1 className="page-admin-header__title"> Add Product</h1>
                    </div>
                </div>                
                <div className="content-container">
                    <AdminForm
                        onSubmit={this.onSubmit}
                    />
                </div>
            </div>
        );
    };
};

const mapDispatchToProps = (dispatch) => ({
    startAddEcommerce: (ecommerce) => dispatch(startAddEcommerce(ecommerce))
});

export default connect(undefined, mapDispatchToProps)(AdminAddPage);