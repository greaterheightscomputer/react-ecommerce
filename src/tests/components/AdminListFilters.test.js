import React from 'react';
import { shallow } from 'enzyme';
import moment from 'moment';
import { AdminListFilters } from '../../components/AdminListFilters';
import { filters, altFilters } from '../fixtures/filters';

let setTextFilter, sortByDate, sortByAmount, categoryTypeMen, categoryTypeWomen, categoryTypeBoy, categoryTypeGirl, 
    itemTypeBoyFootWear, itemTypeBoyClothe, itemTypeBoyHeadWear, itemTypeBoyUnderWear, itemTypeBoyBag, itemTypeBoyTie, itemTypeBoyBelt, itemTypeBoyJewellery, 
    itemTypeWomenFootWear, itemTypeWomenClothe, itemTypeWomenHeadWear, itemTypeWomenUnderWear, itemTypeWomenBag, itemTypeWomenTie, itemTypeWomenBelt, itemTypeWomenJewellery,
    itemTypeMenFootWear, itemTypeMenClothe, itemTypeMenHeadWear, itemTypeMenUnderWear, itemTypeMenBag, itemTypeMenTie, itemTypeMenBelt, itemTypeMenJewellery,
    itemTypeGirlFootWear, itemTypeGirlClothe, itemTypeGirlHeadWear, itemTypeGirlUnderWear, itemTypeGirlBag, itemTypeGirlTie, itemTypeGirlBelt, itemTypeGirlJewellery,
    setStartDate, setEndDate, wrapper;

beforeEach(()=> {
    setTextFilter=jest.fn();  sortByDate=jest.fn(); sortByAmount=jest.fn(); setStartDate=jest.fn(); setEndDate=jest.fn(); 
    categoryTypeBoy=jest.fn(); categoryTypeGirl=jest.fn(); categoryTypeMen=jest.fn(); categoryTypeWomen=jest.fn();  
    itemTypeBoyFootWear=jest.fn(); itemTypeBoyClothe=jest.fn(); itemTypeBoyHeadWear=jest.fn(); itemTypeBoyUnderWear=jest.fn(); itemTypeBoyBag=jest.fn(); itemTypeBoyTie=jest.fn(); itemTypeBoyBelt=jest.fn(); itemTypeBoyJewellery=jest.fn(); 
    itemTypeGirlFootWear=jest.fn(); itemTypeGirlClothe=jest.fn(); itemTypeGirlHeadWear=jest.fn(); itemTypeGirlUnderWear=jest.fn(); itemTypeGirlBag=jest.fn(); itemTypeGirlTie=jest.fn(); itemTypeGirlBelt=jest.fn(); itemTypeGirlJewellery=jest.fn(); 
    itemTypeMenFootWear=jest.fn(); itemTypeMenClothe=jest.fn(); itemTypeMenHeadWear=jest.fn(); itemTypeMenUnderWear=jest.fn(); itemTypeMenBag=jest.fn(); itemTypeMenTie=jest.fn(); itemTypeMenBelt=jest.fn(); itemTypeMenJewellery=jest.fn(); 
    itemTypeWomenFootWear=jest.fn(); itemTypeWomenClothe=jest.fn(); itemTypeWomenHeadWear=jest.fn(); itemTypeWomenUnderWear=jest.fn(); itemTypeWomenBag=jest.fn(); itemTypeWomenTie=jest.fn(); itemTypeWomenBelt=jest.fn(); itemTypeWomenJewellery=jest.fn(); 
    wrapper = shallow(
        <AdminListFilters 
            filters={filters} setTextFilter={setTextFilter} sortByDate={sortByDate} sortByAmount={sortByAmount} setStartDate={setStartDate} setEndDate={setEndDate}
            categoryTypeMen={categoryTypeMen}  categoryTypeBoy={categoryTypeBoy}  categoryTypeGirl={categoryTypeGirl}  categoryTypeWomen={categoryTypeWomen}
            itemTypeBoyFootWear={itemTypeBoyFootWear} itemTypeBoyClothe={itemTypeBoyClothe} itemTypeBoyHeadWear={itemTypeBoyHeadWear} itemTypeBoyUnderWear={itemTypeBoyUnderWear} itemTypeBoyBag={itemTypeBoyBag} itemTypeBoyTie={itemTypeBoyTie} itemTypeBoyBelt={itemTypeBoyBelt} itemTypeBoyJewellery={itemTypeBoyJewellery} 
            itemTypeWomenFootWear={itemTypeWomenFootWear} itemTypeWomenClothe={itemTypeWomenClothe} itemTypeWomenHeadWear={itemTypeWomenHeadWear} itemTypeWomenUnderWear={itemTypeWomenUnderWear} itemTypeWomenBag={itemTypeWomenBag} itemTypeWomenTie={itemTypeWomenTie} itemTypeWomenBelt={itemTypeWomenBelt} itemTypeWomenJewellery={itemTypeWomenJewellery}
            itemTypeMenFootWear={itemTypeMenFootWear} itemTypeMenClothe={itemTypeMenClothe} itemTypeMenHeadWear={itemTypeMenHeadWear} itemTypeMenUnderWear={itemTypeMenUnderWear} itemTypeMenBag={itemTypeMenBag} itemTypeMenTie={itemTypeMenTie} itemTypeMenBelt={itemTypeMenBelt} itemTypeMenJewellery={itemTypeMenJewellery}
            itemTypeGirlFootWear={itemTypeGirlFootWear} itemTypeGirlClothe={itemTypeGirlClothe} itemTypeGirlHeadWear={itemTypeGirlHeadWear} itemTypeGirlUnderWear={itemTypeGirlUnderWear} itemTypeGirlBag={itemTypeGirlBag} itemTypeGirlTie={itemTypeGirlTie} itemTypeGirlBelt={itemTypeGirlBelt} itemTypeGirlJewellery={itemTypeGirlJewellery}            
        />
        );
});

test('should render AdminListFilters with default data', () => {
    expect(wrapper).toMatchSnapshot();
});
test('should render AdminListFilters with data', () => {
    wrapper.setProps({
        filters: altFilters
    });
    expect(wrapper).toMatchSnapshot();
});
test('should set text change', () => {
    const value = 'boy cap';
    wrapper.find('input').simulate('change', { target: { value }});
    expect(setTextFilter).toHaveBeenLastCalledWith(value);
});
test('should sort by date', () => {
    const value = 'date';
    wrapper.setProps({
        filters: altFilters
    });
    wrapper.find('select').at(0).simulate('change', {target: { value }});
    expect(sortByDate).toHaveBeenCalled();
});
test('should sort by amount', () => {
    const value = 'amount';    
    wrapper.find('select').at(0).simulate('change', {target: { value }});
    expect(sortByAmount).toHaveBeenCalled();
});
test('should set date range picker', () => {
    const startDate = moment(0).add(2, 'years');
    const endDate = moment(0).add(4, 'years');
    wrapper.find('DateRangePicker').prop('onDatesChange')({ startDate, endDate });
    expect(setStartDate).toHaveBeenLastCalledWith(startDate);
    expect(setEndDate).toHaveBeenLastCalledWith(endDate);
});
test('should set date focus', () => {
    const focused = 'startDate';    
    wrapper.find('DateRangePicker').prop('onFocusChange')(focused);
    expect(wrapper.state('focused')).toBe(focused);    
});
//category
test('should set category type for men', () => {
    const value = 'amen';    
    wrapper.find('select').at(1).simulate('change', {target: { value }});
    expect(categoryTypeMen).toHaveBeenCalled();
});
test('should set category type for women', () => {
    const value = 'women';    
    wrapper.find('select').at(1).simulate('change', {target: { value }});
    expect(categoryTypeWomen).toHaveBeenCalled();
});
test('should set category type for boy', () => {
    const value = 'boy';    
    wrapper.find('select').at(1).simulate('change', {target: { value }});
    expect(categoryTypeBoy).toHaveBeenCalled();
});
test('should set category type for girl', () => {
    const value = 'girl';    
    wrapper.find('select').at(1).simulate('change', {target: { value }});
    expect(categoryTypeGirl).toHaveBeenCalled();
});
//item - girl
test('should set item type footwear for girl', () => {
    const value = 'girl_footwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeGirlFootWear).toHaveBeenCalled();
});        
test('should set item type clothe for girl', () => {
    const value = 'girl_clothe';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeGirlClothe).toHaveBeenCalled();
});        
test('should set item type headwear for girl', () => {
    const value = 'girl_headwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeGirlHeadWear).toHaveBeenCalled();
});       
test('should set item type underwear for girl', () => {
    const value = 'girl_underwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeGirlUnderWear).toHaveBeenCalled();
});        
test('should set item type bag for girl', () => {
    const value = 'girl_bag';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeGirlBag).toHaveBeenCalled();
});  
test('should set item type tie for girl', () => {
    const value = 'girl_tie';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeGirlTie).toHaveBeenCalled();
});    
test('should set item type belt for girl', () => {
    const value = 'girl_belt';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeGirlBelt).toHaveBeenCalled();
}); 
test('should set item type belt for girl', () => {
    const value = 'girl_jewellery';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeGirlJewellery).toHaveBeenCalled();
});        
//item - boy
test('should set item type footwear for boy', () => {
    const value = 'boy_footwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeBoyFootWear).toHaveBeenCalled();
});        
test('should set item type clothe for boy', () => {
    const value = 'boy_clothe';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeBoyClothe).toHaveBeenCalled();
});        
test('should set item type headwear for boy', () => {
    const value = 'boy_headwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeBoyHeadWear).toHaveBeenCalled();
});       
test('should set item type underwear for boy', () => {
    const value = 'boy_underwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeBoyUnderWear).toHaveBeenCalled();
});        
test('should set item type bag for boy', () => {
    const value = 'boy_bag';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeBoyBag).toHaveBeenCalled();
});  
test('should set item type tie for boy', () => {
    const value = 'boy_tie';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeBoyTie).toHaveBeenCalled();
});    
test('should set item type belt for boy', () => {
    const value = 'boy_belt';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeBoyBelt).toHaveBeenCalled();
}); 
test('should set item type belt for boy', () => {
    const value = 'boy_jewellery';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeBoyJewellery).toHaveBeenCalled();
});        
//item - men
test('should set item type footwear for men', () => {
    const value = 'men_footwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeMenFootWear).toHaveBeenCalled();
});        
test('should set item type clothe for men', () => {
    const value = 'men_clothe';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeMenClothe).toHaveBeenCalled();
});        
test('should set item type headwear for men', () => {
    const value = 'men_headwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeMenHeadWear).toHaveBeenCalled();
});       
test('should set item type underwear for men', () => {
    const value = 'men_underwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeMenUnderWear).toHaveBeenCalled();
});        
test('should set item type bag for men', () => {
    const value = 'men_bag';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeMenBag).toHaveBeenCalled();
});  
test('should set item type tie for men', () => {
    const value = 'men_tie';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeMenTie).toHaveBeenCalled();
});    
test('should set item type belt for men', () => {
    const value = 'men_belt';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeMenBelt).toHaveBeenCalled();
}); 
test('should set item type belt for men', () => {
    const value = 'men_jewellery';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeMenJewellery).toHaveBeenCalled();
});      
//item - women
test('should set item type footwear for women', () => {
    const value = 'women_footwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeWomenFootWear).toHaveBeenCalled();
});        
test('should set item type clothe for women', () => {
    const value = 'women_clothe';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeWomenClothe).toHaveBeenCalled();
});        
test('should set item type headwear for women', () => {
    const value = 'women_headwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeWomenHeadWear).toHaveBeenCalled();
});       
test('should set item type underwear for women', () => {
    const value = 'women_underwear';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeWomenUnderWear).toHaveBeenCalled();
});        
test('should set item type bag for women', () => {
    const value = 'women_bag';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeWomenBag).toHaveBeenCalled();
});  
test('should set item type tie for women', () => {
    const value = 'women_tie';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeWomenTie).toHaveBeenCalled();
});    
test('should set item type belt for women', () => {
    const value = 'women_belt';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeWomenBelt).toHaveBeenCalled();
}); 
test('should set item type belt for women', () => {
    const value = 'women_jewellery';    
    wrapper.find('select').at(2).simulate('change', {target: { value }});
    expect(itemTypeWomenJewellery).toHaveBeenCalled();
});      