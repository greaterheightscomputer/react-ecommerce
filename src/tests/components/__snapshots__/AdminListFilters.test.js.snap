// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AdminListFilters with data 1`] = `
<div
  className="content-container"
>
  <div
    className="input-group"
  >
    <div
      className="input-group__item"
    >
      <input
        className="text-input"
        onChange={[Function]}
        placeholder="Search products"
        type="text"
        value="men shoe"
      />
    </div>
    <div
      className="input-group__item"
    >
      <select
        className="select"
        onChange={[Function]}
        value="amount"
      >
        <option
          value="date"
        >
          Date
        </option>
        <option
          value="amount"
        >
          Amount
        </option>
      </select>
    </div>
    <div
      className="input-group__item"
    >
      <select
        className="select"
        onChange={[Function]}
        value="men"
      >
        <option
          value=""
        >
          Select Category
        </option>
        <option
          value="boy"
        >
          Boy
        </option>
        <option
          value="girl"
        >
          Girl
        </option>
        <option
          value="amen"
        >
          Men
        </option>
        <option
          value="women"
        >
          Women
        </option>
      </select>
    </div>
    <div
      className="input-group__item"
    >
      <select
        className="select"
        onChange={[Function]}
        value="men_footwear"
      >
        <option
          value=""
        >
          Select Item
        </option>
        <option
          value="boy_bag"
        >
          Boy Bag
        </option>
        <option
          value="boy_belt"
        >
          Boy Belt
        </option>
        <option
          value="boy_clothe"
        >
          Boy Clothe
        </option>
        <option
          value="boy_footwear"
        >
          Boy Footwear
        </option>
        <option
          value="boy_headwear"
        >
          Boy Headwear
        </option>
        <option
          value="boy_jewellery"
        >
          Boy Jewellery
        </option>
        <option
          value="boy_tie"
        >
          Boy Tie
        </option>
        <option
          value="boy_underwear"
        >
          Boy Underwear
        </option>
        <option
          value="girl_bag"
        >
          Girl Bag
        </option>
        <option
          value="girl_belt"
        >
          Girl Belt
        </option>
        <option
          value="girl_clothe"
        >
          Girl Clothe
        </option>
        <option
          value="girl_footwear"
        >
          Girl Footwear
        </option>
        <option
          value="girl_headwear"
        >
          Girl Headwear
        </option>
        <option
          value="girl_jewellery"
        >
          Girl Jewellery
        </option>
        <option
          value="girl_tie"
        >
          Girl Tie
        </option>
        <option
          value="girl_underwear"
        >
          Girl Underwear
        </option>
        <option
          value="men_bag"
        >
          Men Bag
        </option>
        <option
          value="men_belt"
        >
          Men Belt
        </option>
        <option
          value="men_clothe"
        >
          Men Clothe
        </option>
        <option
          value="men_footwear"
        >
          Men Footwear
        </option>
        <option
          value="men_headwear"
        >
          Men Headwear
        </option>
        <option
          value="men_jewellery"
        >
          Men Jewellery
        </option>
        <option
          value="men_tie"
        >
          Men Tie
        </option>
        <option
          value="men_underwear"
        >
          Men Underwear
        </option>
        <option
          value="wommen_bag"
        >
          Women Bag
        </option>
        <option
          value="wommen_belt"
        >
          Women Belt
        </option>
        <option
          value="wommen_clothe"
        >
          Women Clothe
        </option>
        <option
          value="wommen_footwear"
        >
          Women Footwear
        </option>
        <option
          value="wommen_headwear"
        >
          Women Headwear
        </option>
        <option
          value="wommen_jewellery"
        >
          Women Jewellery
        </option>
        <option
          value="wommen_tie"
        >
          Women Tie
        </option>
        <option
          value="wommen_underwear"
        >
          Women Underwear
        </option>
      </select>
    </div>
  </div>
  <div
    className="input-group"
  >
    <div
      className="input-group__item"
    >
      <DateRangePicker
        anchorDirection="left"
        customArrowIcon={null}
        customCloseIcon={null}
        customInputIcon={null}
        daySize={39}
        disabled={false}
        displayFormat={[Function]}
        enableOutsideDays={false}
        endDate={"1970-01-03T00:00:00.000Z"}
        endDateId="endDate"
        endDatePlaceholderText="End Date"
        firstDayOfWeek={null}
        focusedInput={null}
        hideKeyboardShortcutsPanel={false}
        horizontalMargin={0}
        initialVisibleMonth={null}
        isDayBlocked={[Function]}
        isDayHighlighted={[Function]}
        isOutsideRange={[Function]}
        isRTL={false}
        keepOpenOnDateSelect={false}
        minimumNights={1}
        monthFormat="MMMM YYYY"
        navNext={null}
        navPrev={null}
        numberOfMonths={1}
        onClose={[Function]}
        onDatesChange={[Function]}
        onFocusChange={[Function]}
        onNextMonthClick={[Function]}
        onPrevMonthClick={[Function]}
        orientation="horizontal"
        phrases={
          Object {
            "chooseAvailableEndDate": [Function],
            "chooseAvailableStartDate": [Function],
            "clearDates": "Clear Dates",
            "closeDatePicker": "Close",
            "dateIsUnavailable": [Function],
            "enterKey": "Enter key",
            "escape": "Escape key",
            "focusStartDate": "Interact with the calendar and add the check-in date for your trip.",
            "hideKeyboardShortcutsPanel": "Close the shortcuts panel",
            "homeEnd": "Home and end keys",
            "jumpToNextMonth": "Move forward to switch to the next month",
            "jumpToPrevMonth": "Move backward to switch to the previous month",
            "keyboardNavigationInstructions": "Press the down arrow key to interact with the calendar and
            select a date. Press the question mark key to get the keyboard shortcuts for changing dates.",
            "keyboardShortcuts": "Keyboard Shortcuts",
            "leftArrowRightArrow": "Right and left arrow keys",
            "moveFocusByOneDay": "Move backward (left) and forward (right) by one day",
            "moveFocusByOneMonth": "Switch months",
            "moveFocusByOneWeek": "Move backward (up) and forward (down) by one week",
            "moveFocustoStartAndEndOfWeek": "Go to the first or last day of a week",
            "openThisPanel": "Open this panel",
            "pageUpPageDown": "page up and page down keys",
            "questionMark": "Question mark",
            "returnFocusToInput": "Return to the date input field",
            "selectFocusedDate": "Select the date in focus",
            "showKeyboardShortcutsPanel": "Open the keyboard shortcuts panel",
            "upArrowDownArrow": "up and down arrow keys",
          }
        }
        readOnly={false}
        renderCalendarInfo={null}
        renderDay={null}
        renderMonth={null}
        reopenPickerOnClearDates={false}
        required={false}
        screenReaderInputMessage=""
        showClearDates={true}
        showDefaultInputIcon={false}
        startDate={"1970-01-01T00:00:00.000Z"}
        startDateId="startDate"
        startDatePlaceholderText="Start Date"
        withFullScreenPortal={false}
        withPortal={false}
      />
    </div>
  </div>
</div>
`;

exports[`should render AdminListFilters with default data 1`] = `
<div
  className="content-container"
>
  <div
    className="input-group"
  >
    <div
      className="input-group__item"
    >
      <input
        className="text-input"
        onChange={[Function]}
        placeholder="Search products"
        type="text"
        value=""
      />
    </div>
    <div
      className="input-group__item"
    >
      <select
        className="select"
        onChange={[Function]}
        value="date"
      >
        <option
          value="date"
        >
          Date
        </option>
        <option
          value="amount"
        >
          Amount
        </option>
      </select>
    </div>
    <div
      className="input-group__item"
    >
      <select
        className="select"
        onChange={[Function]}
        value=""
      >
        <option
          value=""
        >
          Select Category
        </option>
        <option
          value="boy"
        >
          Boy
        </option>
        <option
          value="girl"
        >
          Girl
        </option>
        <option
          value="amen"
        >
          Men
        </option>
        <option
          value="women"
        >
          Women
        </option>
      </select>
    </div>
    <div
      className="input-group__item"
    >
      <select
        className="select"
        onChange={[Function]}
        value=""
      >
        <option
          value=""
        >
          Select Item
        </option>
        <option
          value="boy_bag"
        >
          Boy Bag
        </option>
        <option
          value="boy_belt"
        >
          Boy Belt
        </option>
        <option
          value="boy_clothe"
        >
          Boy Clothe
        </option>
        <option
          value="boy_footwear"
        >
          Boy Footwear
        </option>
        <option
          value="boy_headwear"
        >
          Boy Headwear
        </option>
        <option
          value="boy_jewellery"
        >
          Boy Jewellery
        </option>
        <option
          value="boy_tie"
        >
          Boy Tie
        </option>
        <option
          value="boy_underwear"
        >
          Boy Underwear
        </option>
        <option
          value="girl_bag"
        >
          Girl Bag
        </option>
        <option
          value="girl_belt"
        >
          Girl Belt
        </option>
        <option
          value="girl_clothe"
        >
          Girl Clothe
        </option>
        <option
          value="girl_footwear"
        >
          Girl Footwear
        </option>
        <option
          value="girl_headwear"
        >
          Girl Headwear
        </option>
        <option
          value="girl_jewellery"
        >
          Girl Jewellery
        </option>
        <option
          value="girl_tie"
        >
          Girl Tie
        </option>
        <option
          value="girl_underwear"
        >
          Girl Underwear
        </option>
        <option
          value="men_bag"
        >
          Men Bag
        </option>
        <option
          value="men_belt"
        >
          Men Belt
        </option>
        <option
          value="men_clothe"
        >
          Men Clothe
        </option>
        <option
          value="men_footwear"
        >
          Men Footwear
        </option>
        <option
          value="men_headwear"
        >
          Men Headwear
        </option>
        <option
          value="men_jewellery"
        >
          Men Jewellery
        </option>
        <option
          value="men_tie"
        >
          Men Tie
        </option>
        <option
          value="men_underwear"
        >
          Men Underwear
        </option>
        <option
          value="wommen_bag"
        >
          Women Bag
        </option>
        <option
          value="wommen_belt"
        >
          Women Belt
        </option>
        <option
          value="wommen_clothe"
        >
          Women Clothe
        </option>
        <option
          value="wommen_footwear"
        >
          Women Footwear
        </option>
        <option
          value="wommen_headwear"
        >
          Women Headwear
        </option>
        <option
          value="wommen_jewellery"
        >
          Women Jewellery
        </option>
        <option
          value="wommen_tie"
        >
          Women Tie
        </option>
        <option
          value="wommen_underwear"
        >
          Women Underwear
        </option>
      </select>
    </div>
  </div>
  <div
    className="input-group"
  >
    <div
      className="input-group__item"
    >
      <DateRangePicker
        anchorDirection="left"
        customArrowIcon={null}
        customCloseIcon={null}
        customInputIcon={null}
        daySize={39}
        disabled={false}
        displayFormat={[Function]}
        enableOutsideDays={false}
        endDate={"1970-01-03T00:00:00.000Z"}
        endDateId="endDate"
        endDatePlaceholderText="End Date"
        firstDayOfWeek={null}
        focusedInput={null}
        hideKeyboardShortcutsPanel={false}
        horizontalMargin={0}
        initialVisibleMonth={null}
        isDayBlocked={[Function]}
        isDayHighlighted={[Function]}
        isOutsideRange={[Function]}
        isRTL={false}
        keepOpenOnDateSelect={false}
        minimumNights={1}
        monthFormat="MMMM YYYY"
        navNext={null}
        navPrev={null}
        numberOfMonths={1}
        onClose={[Function]}
        onDatesChange={[Function]}
        onFocusChange={[Function]}
        onNextMonthClick={[Function]}
        onPrevMonthClick={[Function]}
        orientation="horizontal"
        phrases={
          Object {
            "chooseAvailableEndDate": [Function],
            "chooseAvailableStartDate": [Function],
            "clearDates": "Clear Dates",
            "closeDatePicker": "Close",
            "dateIsUnavailable": [Function],
            "enterKey": "Enter key",
            "escape": "Escape key",
            "focusStartDate": "Interact with the calendar and add the check-in date for your trip.",
            "hideKeyboardShortcutsPanel": "Close the shortcuts panel",
            "homeEnd": "Home and end keys",
            "jumpToNextMonth": "Move forward to switch to the next month",
            "jumpToPrevMonth": "Move backward to switch to the previous month",
            "keyboardNavigationInstructions": "Press the down arrow key to interact with the calendar and
            select a date. Press the question mark key to get the keyboard shortcuts for changing dates.",
            "keyboardShortcuts": "Keyboard Shortcuts",
            "leftArrowRightArrow": "Right and left arrow keys",
            "moveFocusByOneDay": "Move backward (left) and forward (right) by one day",
            "moveFocusByOneMonth": "Switch months",
            "moveFocusByOneWeek": "Move backward (up) and forward (down) by one week",
            "moveFocustoStartAndEndOfWeek": "Go to the first or last day of a week",
            "openThisPanel": "Open this panel",
            "pageUpPageDown": "page up and page down keys",
            "questionMark": "Question mark",
            "returnFocusToInput": "Return to the date input field",
            "selectFocusedDate": "Select the date in focus",
            "showKeyboardShortcutsPanel": "Open the keyboard shortcuts panel",
            "upArrowDownArrow": "up and down arrow keys",
          }
        }
        readOnly={false}
        renderCalendarInfo={null}
        renderDay={null}
        renderMonth={null}
        reopenPickerOnClearDates={false}
        required={false}
        screenReaderInputMessage=""
        showClearDates={true}
        showDefaultInputIcon={false}
        startDate={"1970-01-01T00:00:00.000Z"}
        startDateId="startDate"
        startDatePlaceholderText="Start Date"
        withFullScreenPortal={false}
        withPortal={false}
      />
    </div>
  </div>
</div>
`;
